### PR TITLE
Misc feature doc updates for 7.0 related to ASP.NET Core.

### PR DESCRIPTION
**Pull request by biaol-odata**
### Issues

*This pull request fixes issue #1485.*

### Description

*Updates for GH pages related to HttpConfiguration, HttpRequestMessage, routing conventions, and minor cleanups.*

### Checklist (Uncheck if it is not completed)

- [ ] *Test cases added*
- [ ] *Build and test with one-click build and test script passed*

### Additional work necessary

*If documentation update is needed, please add "Docs Needed" label to the issue and provide details about the required document change in the issue.*

### CODE REVIEW

File: _posts/2015-07-24-10-02-work-around-for-singleresult-create.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ layout: post
 category: "10. Others"
 ---
 
+**Note: This work around is for https://github.com/OData/WebApi/issues/170, which and is not applicable for Microsoft.AspNetCore.OData v7.x.**
```
Comment (AlanWong-MS) [inline _posts/2015-07-24-10-02-work-around-for-singleresult-create.md]: typo: which and is -> which is

File: _posts/2015-09-16-04-20-ContinueOnError.md [hunk 2]
```diff
@@ -9,11 +9,27 @@ Since OData Web API V5.7, it supports <strong>[odata.continue-on-error](http://d
...
+
+- For Microsoft.AspNetCore.OData (supporting ASP.NET Core):
+
+   It can be enabled in the service's HTTP request pipeline configuration method `Configure(IApplicationBuilder app, IHostingEnvironment env)` of the typical `Startup` class:
+
+    {% highlight csharp %}
+        app.UseMvc(routeBuilder =>
+        {
+           routeBuilder.Select().Expand().Filter().OrderBy().MaxTop(100).Count()
+                        .EnableContinueOnErrorHeader();	 // Additional configuration to enable continue on error.
```
Comment (AlanWong-MS) [inline _posts/2015-09-16-04-20-ContinueOnError.md]: Is the tab (\t) between the code and comment intended?

File: _posts/2015-09-16-04-19-odataIncludeAnnotations.md [hunk 3]
```diff
@@ -39,7 +39,7 @@ The response will have all annotations:
   "@Hello.World":"Hello World.",
   "PerId":2,
   "Property.BirthdayAnnotation@odata.type":"#DateTimeOffset",
-  "Age@Property.BirthdayAnnotation":"2010-01-02T00:00:00+08:00",
+  "@Property.BirthdayAnnotation":"2010-01-02T00:00:00+08:00",
```
Comment (xuzhg) [inline _posts/2015-09-16-04-19-odataIncludeAnnotations.md]: why remove Age?

File: _posts/2017-12-21-14-01-netcore-beta1.md [hunk 4]
```diff
@@ -143,9 +144,43 @@ As mentioned earlier, most of the samples for Web API OData V6.0.0 apply to Web
...
 
-### OData V7.0.0 for ASP.NET Core 2.x
-
-There is also a new Web API OData V7.0.0 for ASP.NET available as an Alpha, which you can access by downloading the packages
-using this url: [https://www.myget.org/F/webapinetcore/api/v3/index.json](https://www.myget.org/F/webapinetcore/api/v3/index.json).
-At this point, it contains the same feature set as [Web API OData V6.0.0](https://www.nuget.org/packages/Microsoft.AspNet.OData/6.0.0)
-but uses a different namespace: Microsoft.AspNet.OData. The API is the same except for the namespace.
+##### ODataRoutingConvention
+Both `Microsoft.AspNet.OData` and `Microsoft.AspNetCore.OData` packages support same set of OData routing conventions, including default built-in routing conventions and attribute rounting convention, so that each request can be routed to matching controller for processing. All routing conventions implement the interface `IODataRoutingConvention`, however, with different definitions, as highlighted below, for the two packages due to different route matching implementations based on ASP.NET Framework and ASP.NET Core:
+- For ASP.NET Framework:
```
Comment (xuzhg) [inline _posts/2017-12-21-14-01-netcore-beta1.md]: Thank for updating the routing.

Do you have plan to add/update the formatter?

File: _posts/2015-09-16-04-19-odataIncludeAnnotations.md (hunk 3 above)
Comment (biaol-odata) [reply _posts/2015-09-16-04-19-odataIncludeAnnotations.md]: I think "@Property.BirthdayAnnotation" is the annotation name, and 'Age' is the typo. Isn't it?

---
In reply to: [196252720](https://github.com/OData/WebApi/pull/1486#discussion_r196252720) [](ancestors = 196252720)

File: _posts/2017-12-21-14-01-netcore-beta1.md (hunk 4 above)
Comment (biaol-odata) [reply _posts/2017-12-21-14-01-netcore-beta1.md]: I think we can add that later as needed?

---
In reply to: [196253266](https://github.com/OData/WebApi/pull/1486#discussion_r196253266) [](ancestors = 196253266)

**Comment by biaol-odata** [comment]
Merged.

**Pull request merged**